git.go

```diff
@@ -363,9 +364,33 @@ func (s *GitRepo) Ping() bool {
 	return err == nil
 }
 
+// EscapePathSeparator escapes the path separator by replacing it with several.
+// Note: this is harmless on Unix, and needed on Windows.
+func EscapePathSeparator(path string) (string) {
+	switch runtime.GOOS {
+	case `windows`:
+		//On Windows, triple all path separators.
+		//Needed to escape backslash(s) preceding doublequotes,
+		//because of how Windows strings treats backslash+doublequote combo,
+		//and Go seems to be implicitly passing around a doublequoted string on Windows,
+		//so we cannnot use default string instead.
+		//See: https://blogs.msdn.microsoft.com/twistylittlepassagesallalike/2011/04/23/everyone-quotes-command-line-arguments-the-wrong-way/
+		//e.g., C:\foo\bar\ -> C:\\\foo\\\bar\\\
+		//used with --prefix, like this: --prefix=C:\foo\bar\ -> --prefix=C:\\\foo\\\bar\\\
```

Comment:
Hey @shurcooL, Good catch, and thanks for a nice reference I can point *my* devs to! :)